lms/templates/dashboard/course.underscore

```diff
@@ -0,0 +1,643 @@
+<% if (!!settings.enable_verified_certificates) {
+
+        var modeClass = model.course_verified_certs.display_mode;
+
+        if (modeClass != ''){
+            modeClass = ' ' + modeClass ;
+        } else {
+            modeClass = '';
+        }
+    }
+    var courseTarget = _.sprintf( '/courses/%(courseKey)s/info', { courseKey: model.course_key });
```

Comment:
This is just a string so please define it in the View and pass it into the template.